src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -130,6 +130,18 @@ public IActionResult GetUserFollowerById(Guid id)
             return Ok(listOfFollowedUsers);
         }
 
+        [HttpGet]
+        [Route("post/{id}")]
+        public IActionResult PostedItem(Guid id)
+        {
+            var post = _context.Posts.Find(id);
+
+            if(post == null)
+            {
+                return BadRequest();
+            }
+            return Ok(post.Content);
```

Comment:
this is supposed to return the entire post object rather than only the text content. Please fix

Reply:
Ok.done